src/features/People/PeoplePage/Content/index.js

```diff
@@ -47,7 +47,7 @@ const Content = () => {
                             <PersonCard
                                 key={person.id}
                                 link={person.id}
-                                pictureUrl={`https://image.tmdb.org/t/p/w400${person.profile_path}`}
+                                pictureUrl={person.profile_path? `https://image.tmdb.org/t/p/w400${person.profile_path}` : null}
```

Comment:
daj jeszcze spację przed `?`, i fajnie by było jakbyś link `https://image.tmdb.org/t/p/w400` wrzucił do configAPi i importował gdzie trzeba, żeby w razie czego można było zmienić go w jednym miejscu 😁można też podzielić na 2 linki - w400 albo original (bo chyba tylko takie używamy).

Reply:
a sorki, właśnie zauważyłem że z tymi linkami to nie Twoje zadanie 😅